modules/microshift-cni.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-cni_{context}"]
+= About the OVN-Kubernetes CNI network provider
+
+//Q: this module is not done; included for the sake of the build
+//[IMPORTANT]
+//====
+//The default CNI configuration is intended to match the developer environment described in MicroShift Development Environment on RHEL 8.
+//====
+//Q: What's the default production configuration?
```

Comment:
It should be the same. What gave you the impression it would be different?

Reply:
Installation changes. I thought we didn't want users to have to use VMs; I thought that was what this was referring to.